pkg/api/v1/types.go

```diff
@@ -1631,7 +1631,10 @@ type Endpoints struct {
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
 	ObjectMeta `json:"metadata,omitempty"`
 
-	// The set of all endpoints is the union of all subsets.
+	// The set of all endpoints is the union of all subsets. Addresses are placed into
+	// subsets according to the IPs they share. If a single address has multiple ports,
```

Comment:
What does it mean for a port to be ready?  AFAIU readiness is a property of the pod, not of a pod-port?